index.bs

```diff
@@ -73,7 +72,17 @@ Definitions {#definitions}
 ==========================
 
 : <dfn>Dataset</dfn>
-:: A collection of data. A dataset is identified with at least its name and [=publisher=].
+:: A collection of data.
+    A set of metadata that includes the dataset’s name and [=publisher=].
+    The data objects themselves are not part of the dataset proper but provided in its [=distributions=].
+    
+    For example: imagine a dataset of Van Gogh paintings called ‘Sunflowers’,
+    which is published by the Van Gogh Museum under a specific license.
+    These are all part of the dataset.
+    The dataset also tells us the URLs of distributions where we can download or query the data.
+    If we consult the data provided by one of these distributions, we’ve left the sphere of the dataset proper.
+    That is to say, the data itself, which may include descriptions of paintings, persons and places,
+    are *not* properties of the dataset.
```

Comment:
Does this make sense?

Reply:
I known you mean _dataset description_ (or _dataset metadata_) when you say _dataset_, but this might not be clear to every reader. May add a sentence somewhere in the req-doc where we make this distinction, so readers have the same understanding?